lib/consoletest.pm

```diff
@@ -27,6 +27,13 @@ sub post_fail_hook {
     upload_logs "/tmp/full_journal.log";
     assert_script_run("dmesg > /tmp/dmesg.log");
     upload_logs "/tmp/dmesg.log";
+    # Export extra log after failure for further check gdm issue 1127317, also poo#45236 used for tracking action on Openqa
+    assert_script_run("tar -jcv -f /tmp/xorg.tar.bz2  /home/bernhard/.local/share/xorg");
```

Comment:
Please don't use assert_script_run since maybe some file depend on yourself setting will cause it create tar file fail, and should check whether the upload_logs will return failure when the tar file doesn't exist.